experimental/segmenter/src/math_helper.rs

```diff
@@ -3,6 +3,8 @@
 // (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 
 use ndarray::{concatenate, Array1, Array2, ArrayBase, Axis, Dim, ViewRepr};
+#[allow(unused_imports)]
```

Comment:
Question: why is it seen as an unused_import? The compiler should pick up that you're calling trait methods I think.

Reply:
This is an extension trait. If `std` is available, this will be shadowed and `v.exp()` will be `f32::exp` and not `num_traits::Float::exp`.

Reply:
We can't gate on std actually being there or not, and the `std` feature isn't always accurate, e.g. with `--features datagen` std will be enabled due to some dependencies, but the `std` feature will not.

Reply:
OK.

Nit: Add a code comment explaining this.